lib/Controller/PageController.php

```diff
@@ -122,21 +126,36 @@ public function indexFPath(): TemplateResponse {
 	 */
 	#[NoAdminRequired]
 	#[NoCSRFRequired]
+	#[RequireSetupOk]
 	#[PublicPage]
 	#[RequireSignRequestUuid]
 	public function sign($uuid): TemplateResponse {
 		$this->initialState->provideInitialState('action', JSActions::ACTION_SIGN);
-		$this->initialState->provideInitialState('config', array_merge(
-			$this->accountService->getConfig($this->userSession->getUser()),
+		$this->initialState->provideInitialState('config',
```

Comment:
This code section are really confusing and in others areas that we are using for provide initial state. Let's consider to add some design patterns here to split this code.

Reply:
Maybe after. At this moment we need to think more about if is necessary or not or also if we will split again the config.

For now the intention is only to reduce the responsibility of config method.